h/accounts/services.py

```diff
@@ -115,6 +115,16 @@ def login(self, username_or_email, password):
 
         return None
 
+    def annotation_count(self, userid):
+        """
+        Return the count of shared annotations for this user.
+        """
+
+        return (
+            self.session.query(Annotation)
+            .filter_by(userid=userid, shared=True)
```

Comment:
Ah! One thing I totally missed in the last review (sorry), but just noticed now when I tested is that we have to filter by `groupid='__world__'` as well in this case. As we don't want to expose how many annotations a user made in other groups. This also means that the test should create a bunch of annotations in a group that is not `__world__` to test that we don't count them.